airbyte-webapp/.stylelintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-standard-scss",
+    "stylelint-config-css-modules",
+    "stylelint-config-prettier-scss"
+  ],
+  "rules": {
+    "selector-class-pattern": null,
+    "color-function-notation": null,
+    "font-family-name-quotes": null,
+    "scss/dollar-variable-empty-line-before": null,
+    "scss/dollar-variable-pattern": ".*",
+    "scss/percent-placeholder-pattern": ".*",
+    "value-keyword-case": null
+  },
+  "ignoreFiles": ["**/build/**", "**/dist/**"]
```

Comment:
Does stylelint have a way to maybe configure includes instead, to just have src and public included? (I see us totally forgetting this path if we'd later change build output directories).

Reply:
[This](https://stylelint.io/user-guide/configure/#ignorefiles) implies theres a `files` config but I don't see any documentation for it.

Reply:
In most cases we use `camelCase` for class names, so probably we need to configure the pattern for `"selector-class-pattern"`?

Reply:
Added!